staff/alex-barba/spa/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```

Comment:
You could put a title here, maybe something like "SPA".

Reply:
I just updated ti to "Document", which is the title that I have thought for this great project. Thanks anyway!